src/mcpil.py

```diff
@@ -470,19 +512,22 @@ def multiplayer_tab(parent):
     current_ip = StringVar(main_frame)
     current_ip.trace('w', lambda *args: update_proxy())
     ip = ttk.Entry(main_frame, width=24, textvariable=current_ip)
+    ip.tooltip = ToolTip(ip, text="This is the IP of the multiplayer server.\nThe IP might be a domain, for example, thebrokenrail.com,\nor it might be an IP adress such as 192.168.1.3")
     ip.grid(row=0, column=1, padx=6, pady=6, sticky='EW')
 
```

Comment:
Oh,right, I'm on an eBook and as an admin you can change stuff in my fork, so if you want to change stuff, change!

Reply:
?

Reply:
I didn’t request changes to this